docker/docker-compose.yml

```diff
@@ -574,8 +574,8 @@ services:
     image: mariadb:10.5.21
     container_name: mariadb
     stop_grace_period: 5s
-    ports:
-      - 3306:3306
+    # ports:
+    #   - "127.0.0.1:3306:3306"
```

Comment:
There are a few cases where we expose ports even though they shouldn't be required (because the "client" can access them directly through Docker container networks).

I am not sure if someone is using these ports anyway -- eg to inspect the DB content from the host.

This serves `mempool` and I doubt anyone used it, so I commented it out.

Reply:
Personally I've never used this one, I would be for removing even the comment.